src/foam/u2/tag/Input.js

```diff
@@ -51,6 +53,21 @@ foam.CLASS({
       if ( this.type ) this.attrs({type: this.type});
       this.cssClass(this.myCls());
       this.link();
+      if ( this.autocompleter ) {
```

Comment:
Should these things really be added to the Input tag model?

I feel like we should have a higher level TextFieldView that composes an input tag and hooks up autocomplete and what not.  Otherwise it cuts down on the re-usability of the raw tag.